diagnostic.sql

```diff
@@ -3,13 +3,29 @@ CREATE DATABASE "sql-crud-assessment";
 \pset pager 0
 \c "sql-crud-assessment"
 
--- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
-
--- your code here
+-- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender,
+-- height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.
+-- Don't forget that all tables should have an id column as a unique identifier for a row.
+
+CREATE TABLE licensees (
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
```

Comment:
TEXT isn't exactly wrong, but it's really intended for longer blocks of text, e.g. paragraphs.
CHARACTER VARYING is a better fit for short strings of text. You can even specify a size for CHARACTER VARYING when you know the length of the strings you want to be dealing with.